wiki/layout/subpage-bottom-nav.html

```diff
@@ -7,14 +7,8 @@
 <div class="flex flex-wrap w-full lg:px-20 xl:px-60 justify-center mb-8">
   <template x-for="(sp, index) in subpages" :key="sp.key">
     <template x-if="sp.img != ''">
```

Comment:
This is removing the subpage feature entirely. If you go to human practices / human practices page, you can see those subpages in action below the top navigation bar.

Reply:
Oh sorry! Did not realize that got changed. I've fixed it now so the button is gone and the human practices/human practices page is not affected.